docs/content/en/docs/concepts/support-versions.md

```diff
@@ -61,11 +61,9 @@ Bottlerocket, Ubuntu, and Red Hat Enterprise Linux (RHEL) can be used as operati
 |               | 1.13.1    | 0.15-0.17
 |               | 1.12.0    | 0.14
 |               | 1.10.1    | 0.12
-| RHEL          | 9.x<sup>*</sup>	      | 0.18
+| RHEL          | 9.x	      | 0.18
```

Comment:
We should keep this asterisk and change the below text to "Nutanix and CloudStack only" (we can remove it once we add it for bare metal)